packages/react-dom/src/__tests__/ReactLegacyUpdates-test.js

```diff
@@ -903,37 +903,52 @@ describe('ReactLegacyUpdates', () => {
     let container = document.createElement('div');
     let component = ReactDOM.render(<A />, container);
 
-    expect(() => {
-      expect(() => component.setState({}, 'no')).toErrorDev(
-        'Expected the last optional `callback` argument to be ' +
-          'a function. Instead received: no.',
+    await expect(async () => {
+      await expect(async () => {
+        await act(() => {
+          component.setState({}, 'no');
+        });
+      }).rejects.toThrowError(
+        'Invalid argument passed as callback. Expected a function. Instead ' +
+          'received: no',
       );
-    }).toThrowError(
-      'Invalid argument passed as callback. Expected a function. Instead ' +
-        'received: no',
+    }).toErrorDev(
+      'Expected the last optional `callback` argument to be ' +
+        'a function. Instead received: no.',
+      {withoutStack: 1},
```

Comment:
Many of the test changes here removed the component stacks, is that expected?

Reply:
These are the same. They weren't actually asserting before.

Reply:
Sweet cool cool cool